README.md

```diff
@@ -1,33 +1,305 @@
-# OriginTrail Decentralized Knowledge Graph network node - OT-Node v6
+<a name="readme-top"></a>
 
+___
 
-OriginTrail technology is a set of tools and protocols evolving as the semantic layer for Web3, built on the OriginTrail Decentralized Knowledge Graph (DKG). Its function is to empower developers and system integrators utilizing decentralized knowledge exchange supported by emerging standards and DLT networks.
+<br />
+<div align="center">
+  <a href="https://github.com/OriginTrail/dkg.py">
```

Comment:
Lets fix this link, maybe we can put it to direct to the origintrail.io, ot-node or the docs